src/uwtools/tests/test_schemas.py

```diff
@@ -725,7 +725,7 @@ def test_schema_execution_threads():
 def test_schema_execution_serial():
     config = {"executable": "fv3"}
     batchargs = {"batchargs": {"queue": "string", "walltime": "string"}}
-    errors = schema_validator("execution")
+    errors = schema_validator("execution-serial")
```

Comment:
I saw this was originally `execution`, but feel that it should have been `execution-serial` since it is testing the `execution-serial` schema and not `execution-parallel`.